src/gui/widgets/template_list_widget.cpp

```diff
@@ -806,7 +800,9 @@ void TemplateListWidget::updateButtons()
 		if (temp->isTemplateGeoreferenced())
 		{
 			georef_visible = true;
-			georef_active  = true;
+			// Unsafe to release unloaded template
+			// See comment in Template::releaseGeoref().
+			georef_active = main_view->getTemplateVisibility(temp).visible;
```

Comment:
Don't enable when the row is not checked. (If it is checked, then it is loaded!)
~~~
                   && template_table->item(visited_row, 0)->checkState() == Qt::Checked
~~~

Reply:
That was what I intended, please replace the visibible test with the checkState test.

Reply:
You are right. I was confused by the different implementation in the other branch of the `if` statement.